views/register.ejs

```diff
@@ -17,12 +16,13 @@
     <label name="password1">Confirm your password:</label>
     <input type="password" name="password1" />
   </div>
-  <input type="hidden" name="_csrf" value="<%= _csrf %>">
+  <input type="hidden" name="_csrf" value="<%= _csrf %>" />
```

Comment:
Having the CSRF hidden input twice breaks the CSRF token validation, so you need to remove one of these.

Reply:
removed